Automattic/jetpack
# [Plugin] Backup: Redirect to plugin page on plugin activation.

Whenever the Jetpack Backup plugin is activated, redirect to the Backup plugin page instead of staying on the plugins page. 




#### Changes proposed in this Pull Request:

* On plugin activation, using the `activated_plugin` hook, check if it's the Jetpack Backup plugin, and redirect to the plugin main page. 

#### Does this pull request change what data or activity we track or use?
No

#### Project Asana card:
1201228005200946-as-1201228017959572

#### Testing instructions:
- Apply this PR locally.
- Deactivate and reactivate the Jetpack Backup plugin on your site.
- Confirm that you're taken to the Jetpack Backup page.
